src/ui/topeditorcontainer.cpp

```diff
@@ -82,10 +82,17 @@ EditorTabWidget *TopEditorContainer::tabWidgetFromEditor(Editor *editor)
 
 void TopEditorContainer::on_currentTabChanged(int index)
 {
+    if(index == -1)
```

Comment:
According to the [doc](http://doc.qt.io/qt-5/qtabwidget.html#currentChanged) index is -1 when the last tab gets closed. Funny that this hasn't bitten us in the ass earlier, but we've got to handle that error case because it happens every time a windows is closed completely.